### DISCUSSION TITLE
Daily Spring 03/21

### DISCUSSION THREAD

**Discussion by revature-ives** [algoives/TestBankApp#59] [Daily Scrum]
**What I Did Yesterday**

I started working on the business logic of There quizz select view

**Obstacles**

problems biding the variable in the where close for queries
find the right way to layout image in the collection vie to display a quizz

**What I Going To do tomorrow**

Continue working on the same
Logic of the quizz selection 
need to get the id of the quizz selected

**Comment by ClarkeWilliford** [comment]
https://us05web.zoom.us/j/88956553224?pwd=cVl0a3RTdVZVSGpXVzR5cWxjRWZZUT09

zoom meeting link
